src/systemcmds/tests/test_autodeclination.cpp

```diff
@@ -23,7 +23,7 @@ class AutoDeclinationTest : public UnitTest
 
 bool AutoDeclinationTest::autodeclination_check(void)
 {
-	ut_assert("declination differs more than 1 degree", get_mag_declination(47.0, 8.0) - 0.6f < 0.5f);
+	ut_assert("declination differs more than 1 degree", get_mag_declination(47.0, 8.0) - 1.6f < 0.5f);
```

Comment:
@stephanbro We run this test suite mostly because it runs in SITL, in CI as unit test and on-hardware. Could you add all test cases you added to GTest here as well? GTest might be eventually factored out.